[CI Failure]: Diffusion X2I(&A&T) · Perf Test, test_diffusion_performance_benchmark[0-test_qwen_image_ulysses2_cfg2_vae_patch4], AssertionError: latency_mean: 3.1746 > baseline 2.7

### Your current environment

<details>
<summary>The output of <code>python collect_env.py</code></summary>

```text
ci env
```

</details>


### Your code version

<details>
<summary>The commit id or version of vllm</summary>

```text
0.19.0
```
</details>
<details>
<summary>The commit id or version of vllm-omni</summary>

```text
817e32d548de74d374b34b6f7dcdccb8342cf4cd
```
</details>


### 🐛 Describe the bug

https://buildkite.com/vllm/vllm-omni/builds/6979/steps/canvas?sid=019d9784-73af-48b9-9fd6-ac9e55adc857&tab=output
```
============================================================================ FAILURES ============================================================================

_______________________________________ test_diffusion_performance_benchmark[0-test_qwen_image_ulysses2_cfg2_vae_patch4] _______________________________________

diffusion_server = <run_diffusion_benchmark.DiffusionServer object at 0x76fa3b21f3e0>
benchmark_params = {'params': {'baseline': {'latency_mean': 2.7, 'peak_memory_mb_mean': 61000, 'throughput_qps': 0.1}, 'dataset': 'random', 'enable-negative-prompt': True, 'height': 512, ...}, 'test_name': 'test_qwen_image_ulysses2_cfg2_vae_patch4'}

    @pytest.mark.parametrize(
        "diffusion_server",
        server_params,
        ids=[p["test_name"] for p in server_params],
        indirect=True,
    )
    @pytest.mark.parametrize("benchmark_params", benchmark_indices, indirect=True)
    def test_diffusion_performance_benchmark(diffusion_server, benchmark_params):
        """Run the diffusion performance benchmark and assert against baselines.

        One server is started per unique parallel configuration (module scope).
        For each server, all benchmark parameter sets defined in the config JSON
        are executed sequentially; results are asserted against the baselines.

        Tracked metrics:
            - throughput_qps          (higher is better)
            - latency_p50, latency_p99 (lower is better)
        """
        test_name = benchmark_params["test_name"]
        params = benchmark_params["params"]
        backend = diffusion_server.server_type  # "vllm-omni"

        result = run_benchmark(
            host=diffusion_server.host,
            port=diffusion_server.port,
            model=diffusion_server.model,
            params=params,
            test_name=test_name,
            backend=backend,
            server_cfg=getattr(diffusion_server, "server_cfg", {}),
            source_file=cast(str, CONFIG_FILE_PATH),
        )

        print(f"\n{'='*60}")
        print(f"Results for {test_name} (server={diffusion_server.server_type}, backend={backend}):")
        for key in (
            "throughput_qps",
            "latency_mean",
            "latency_median",
            "latency_p50",
            "latency_p99",
            "peak_memory_mb_max",
            "peak_memory_mb_mean",
            "peak_memory_mb_median",
        ):
            if key in result:
                print(f"  {key}: {result[key]:.4f}")
        print(f"\n  Aggregated results: {AGGREGATED_RESULT_FILE}")
        print("=" * 60)

>       assert_result(result, params)

tests/dfx/perf/scripts/run_diffusion_benchmark.py:710:

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

result = {'backend': 'vllm-omni', 'completed_requests': 10, 'dataset': 'random', 'duration': 31.748929443303496, ...}
params = {'baseline': {'latency_mean': 2.7, 'peak_memory_mb_mean': 61000, 'throughput_qps': 0.1}, 'dataset': 'random', 'enable-negative-prompt': True, 'height': 512, ...}

    def assert_result(result: dict[str, Any], params: dict[str, Any]) -> None:
        """Assert that benchmark metrics satisfy the configured baselines."""
        num_prompts = params.get("num-prompts", 10)
        completed = result.get("completed_requests", result.get("completed", 0))
        assert completed == num_prompts, f"Expected {num_prompts} completed requests, got {completed}"

        for metric, threshold in params.get("baseline", {}).items():
            current = result.get(metric)
            assert current is not None, f"Metric '{metric}' not found in result: {list(result.keys())}"
            if "throughput" in metric:
                assert current >= threshold, f"{metric}: {current:.4f} < baseline {threshold}"
            else:
>               assert current <= threshold, f"{metric}: {current:.4f} > baseline {threshold}"
E               AssertionError: latency_mean: 3.1746 > baseline 2.7
E               assert 3.174646803876385 <= 2.7

tests/dfx/perf/scripts/run_diffusion_benchmark.py:651: AssertionError

======================================================================== warnings summary ========================================================================
<frozen importlib._bootstrap>:488
  <frozen importlib._bootstrap>:488: DeprecationWarning: builtin type SwigPyPacked has no __module__ attribute

<frozen importlib._bootstrap>:488
  <frozen importlib._bootstrap>:488: DeprecationWarning: builtin type SwigPyObject has no __module__ attribute

../../../usr/local/lib/python3.12/dist-packages/torch/jit/_script.py:362: 14 warnings
  /usr/local/lib/python3.12/dist-packages/torch/jit/_script.py:362: DeprecationWarning: `torch.jit.script_method` is deprecated. Please switch to `torch.compile` or `torch.export`.
    warnings.warn(

vllm_omni/__init__.py:28
  /workspace/build/buildkite/vllm_omni/__init__.py:28: RuntimeWarning: Failed to import version from _version.py: No module named 'vllm_omni._version'
  This typically happens in development mode before building.
    from .version import __version__, __version_tuple__  # isort:skip

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html

==================================================================== short test summary info ====================================================================
FAILED tests/dfx/perf/scripts/run_diffusion_benchmark.py::test_diffusion_performance_benchmark[0-test_qwen_image_ulysses2_cfg2_vae_patch4] - AssertionError: latency_mean: 3.1746 > baseline 2.7
assert 3.174646803876385 <= 2.7
==================================================== 1 failed, 7 passed, 17 warnings in 1251.89s (0:20:51) ====================================================
sys:1: DeprecationWarning: builtin type swigvarlink has no __module__ attribute

===================================================================== test session starts ============================
```

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://vllm-omni.readthedocs.io), which can answer lots of frequently asked questions.

--- a/tests/dfx/perf/tests/test_qwen_image_vllm_omni.json
+++ b/tests/dfx/perf/tests/test_qwen_image_vllm_omni.json
@@ -106,22 +106,6 @@
         },
         "benchmark_params": [
             {
-                "name": "512x512_steps20",
-                "dataset": "random",
-                "task": "t2i",
-                "width": 512,
-                "height": 512,
-                "num-inference-steps": 20,
-                "num-prompts": 10,
-                "max-concurrency": 1,
-                "enable-negative-prompt": true,
-                "baseline": {
-                    "throughput_qps": 0.1,
-                    "latency_mean": 2.7,
-                    "peak_memory_mb_mean": 61000
-                }
-            },
-            {
                 "name": "1536x1536_steps35",
                 "dataset": "random",
                 "task": "t2i",

